lib/udf_string_utils.rb

```diff
@@ -244,6 +244,55 @@ class UdfStringUtils
                            {query: "select ?(1, 3, 1, ',')", expect: "1,2" , example: true},
                            {query: "select ?(1, 5, 2, '|')", expect: "1|3", example: true},
                        ]
+      }, {
+          type:        :function,
```

Comment:
Push your changes to the remote and I can take a look, there must be something wrong in your code because what say you did should be enough.